sim/verilator-uhdm/meta.yaml

```diff
@@ -11,29 +13,29 @@ source:
 
 build:
   number: {{ environ.get('DATE_NUM') }}
-  string: {{ environ.get('DATE_STR') }}
+  string: {{ environ.get('DATE_STR') }}_{{ py_suffix }}
   script_env:
     - CI
-    - USE_SYSTEM_GCC_VERSION
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - python {{ python }}
+    - bison
     - cmake
-    - pkg-config
+    - flex
     - libuuid
+    - pkg-config
+  host:
+    - python {{ python_version }}
     - gperftools
-    - flex
     - libunwind
     - swig
   run:
-    - {{ compiler('cxx') }}
```

Comment:
Not sure about bison or flex but don't you need a C++ compiler to use verilator?

Reply:
True, I've added them.